<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>job</class>
 <widget class="QMainWindow" name="job">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1428</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>job</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 0);

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="moneyUpdate">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pool</string>
    </property>
   </widget>
   <widget class="QLabel" name="afzayesheLevel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>afzlevel</string>
    </property>
   </widget>
   <widget class="QLabel" name="moment">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>moment</string>
    </property>
   </widget>
   <widget class="QLabel" name="level">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>level</string>
    </property>
   </widget>
   <widget class="QPushButton" name="timerClick">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 6pt &quot;Wide Latin&quot;;
background-color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>timerClick</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moneyPlus">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 127);
font: 75 6pt &quot;Wide Latin&quot;;</string>
    </property>
    <property name="text">
     <string>level</string>
    </property>
   </widget>
   <widget class="QPushButton" name="timerClick_2">
    <property name="geometry">
     <rect>
      <x>352</x>
      <y>220</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 6pt &quot;Wide Latin&quot;;
background-color: rgb(85, 0, 127);</string>
    </property>
    <property name="text">
     <string>timerClick</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moneyPlus_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 0, 127);
font: 75 6pt &quot;Wide Latin&quot;;</string>
    </property>
    <property name="text">
     <string>level</string>
    </property>
   </widget>
   <widget class="QLabel" name="moneyUpdate_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>140</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pool</string>
    </property>
   </widget>
   <widget class="QLabel" name="moment_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>moment</string>
    </property>
   </widget>
   <widget class="QLabel" name="level_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>level</string>
    </property>
   </widget>
   <widget class="QLabel" name="afzayesheLevel_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>140</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>afzlevel</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 255);</string>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbar_2">
    <property name="geometry">
     <rect>
      <x>397</x>
      <y>280</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 255);</string>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="Daramad">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>80</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>daramad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moneyPlus_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>220</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 127);
font: 75 6pt &quot;Wide Latin&quot;;</string>
    </property>
    <property name="text">
     <string>level</string>
    </property>
   </widget>
   <widget class="QLabel" name="level_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>180</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>level</string>
    </property>
   </widget>
   <widget class="QPushButton" name="timerClick_3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>220</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 127);
font: 75 6pt &quot;Wide Latin&quot;;</string>
    </property>
    <property name="text">
     <string>timerClick</string>
    </property>
   </widget>
   <widget class="QLabel" name="moneyUpdate_3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>140</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pool</string>
    </property>
   </widget>
   <widget class="QLabel" name="afzayesheLevel_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>140</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>afzlevel</string>
    </property>
   </widget>
   <widget class="QLabel" name="moment_3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>180</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>moment</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbar_3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>280</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 255);</string>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="moneyPlus_4">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>220</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 0);
font: 75 6pt &quot;Wide Latin&quot;;</string>
    </property>
    <property name="text">
     <string>level</string>
    </property>
   </widget>
   <widget class="QLabel" name="level_4">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>180</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>level</string>
    </property>
   </widget>
   <widget class="QPushButton" name="timerClick_4">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>220</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 6pt &quot;Wide Latin&quot;;
background-color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>timerClick</string>
    </property>
   </widget>
   <widget class="QLabel" name="moneyUpdate_4">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>140</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pool</string>
    </property>
   </widget>
   <widget class="QLabel" name="afzayesheLevel_4">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>140</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>afzlevel</string>
    </property>
   </widget>
   <widget class="QLabel" name="moment_4">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>180</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>moment</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbar_4">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>280</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 255);</string>
    </property>
    <property name="maximum">
     <number>25</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="moneyPlus_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>500</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 85, 0);
font: 75 6pt &quot;Wide Latin&quot;;</string>
    </property>
    <property name="text">
     <string>level</string>
    </property>
   </widget>
   <widget class="QLabel" name="level_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>460</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>level</string>
    </property>
   </widget>
   <widget class="QPushButton" name="timerClick_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 6pt &quot;Wide Latin&quot;;
background-color: rgb(255, 85, 0);</string>
    </property>
    <property name="text">
     <string>timerClick</string>
    </property>
   </widget>
   <widget class="QLabel" name="moneyUpdate_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pool</string>
    </property>
   </widget>
   <widget class="QLabel" name="afzayesheLevel_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>420</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>afzlevel</string>
    </property>
   </widget>
   <widget class="QLabel" name="moment_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>moment</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbar_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>560</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 255);</string>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="moneyPlus_6">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>500</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);
font: 75 6pt &quot;Wide Latin&quot;;</string>
    </property>
    <property name="text">
     <string>level</string>
    </property>
   </widget>
   <widget class="QLabel" name="level_6">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>460</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>level</string>
    </property>
   </widget>
   <widget class="QPushButton" name="timerClick_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>500</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);
font: 75 6pt &quot;Wide Latin&quot;;</string>
    </property>
    <property name="text">
     <string>timerClick</string>
    </property>
   </widget>
   <widget class="QLabel" name="moneyUpdate_6">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>420</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pool</string>
    </property>
   </widget>
   <widget class="QLabel" name="afzayesheLevel_6">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>420</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>afzlevel</string>
    </property>
   </widget>
   <widget class="QLabel" name="moment_6">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>460</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>moment</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbar_6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>560</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 255);</string>
    </property>
    <property name="maximum">
     <number>35</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="moneyPlus_7">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>500</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 0);
font: 75 6pt &quot;Wide Latin&quot;;</string>
    </property>
    <property name="text">
     <string>level</string>
    </property>
   </widget>
   <widget class="QLabel" name="level_7">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>460</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>level</string>
    </property>
   </widget>
   <widget class="QPushButton" name="timerClick_7">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>500</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 0);
font: 75 6pt &quot;Wide Latin&quot;;</string>
    </property>
    <property name="text">
     <string>timerClick</string>
    </property>
   </widget>
   <widget class="QLabel" name="moneyUpdate_7">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>420</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pool</string>
    </property>
   </widget>
   <widget class="QLabel" name="afzayesheLevel_7">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>420</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>afzlevel</string>
    </property>
   </widget>
   <widget class="QLabel" name="moment_7">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>460</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>moment</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbar_7">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>560</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 255);</string>
    </property>
    <property name="maximum">
     <number>40</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="moneyPlus_8">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>500</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(29, 68, 58);
font: 75 6pt &quot;Wide Latin&quot;;</string>
    </property>
    <property name="text">
     <string>level</string>
    </property>
   </widget>
   <widget class="QLabel" name="level_8">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>460</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>level</string>
    </property>
   </widget>
   <widget class="QPushButton" name="timerClick_8">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>500</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 6pt &quot;Wide Latin&quot;;
background-color: rgb(29, 68, 58);</string>
    </property>
    <property name="text">
     <string>timerClick</string>
    </property>
   </widget>
   <widget class="QLabel" name="moneyUpdate_8">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>420</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pool</string>
    </property>
   </widget>
   <widget class="QLabel" name="afzayesheLevel_8">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>420</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>afzlevel</string>
    </property>
   </widget>
   <widget class="QLabel" name="moment_8">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>460</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>moment</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbar_8">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>560</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 255);</string>
    </property>
    <property name="maximum">
     <number>45</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>470</y>
      <width>131</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 229);</string>
    </property>
    <item>
     <property name="text">
      <string>price_job2=6$</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>price_job3=10$</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>price_job4=15$</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>price_job5=25$</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>price_job6=34$</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>price_job7=40$</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>price_job8=50$</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>1051</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 36pt &quot;Terminal&quot;;</string>
    </property>
    <property name="text">
     <string>WELCOME TO WILDWESTSAGA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>440</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 11pt &quot;8514oem&quot;;
background-color: rgb(131, 150, 255);</string>
    </property>
    <property name="text">
     <string>&quot;PRICES&quot;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1428</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
